auth-web/src/components/auth/common/Product.vue

```diff
@@ -180,23 +184,26 @@ export default class Product extends Vue {
   get showProductFee () {
     return this.canManageProductFee && this.orgProduct && this.orgProduct.product
   }
-
+  private get isStaff (): boolean {
+    return this.currentUser.roles.includes(Role.Staff)
+  }
   get productLabel () {
     // this is mapping product code with lang file.
     // lang file have subtitle and description with product code prefix.
     // eg: pprCodeSubtitle, pprCodeDescription
     // Also, returns check box icon and color if the product has been reviewed.
     const { code } = this.productDetails
-    let subTitle = `${code && code.toLowerCase()}CodeSubtitle` || ''
-    let details = `${code && code.toLowerCase()}CodeDescription` || ''
+    let mhrNotStaff = !this.isStaff && code.toLowerCase() === MhrProductCodeConstants.MHR.toLowerCase() ? MhrProductCodeConstants.NOT_STAFF : ''
```

Comment:
Is there a way to simplify this? From what I see here, we just want to show subtitle and details for non-staff roles. Maybe we could check for that condition and avoid all the default empty strings...

Reply:
Subtitle and details are shown for all products.  

This logic builds a string for selecting from the lang file.  it is only applicable for MHR non staff.

Reply:
I think this would be a lot simpler if you set `code` to `mhrNotStaff` here and remove the MhrProductCodeConstants and all the code changes below. That way if we decide to refactor it later or add other 'staff' versions of descriptions its just one line of code to add to / alter

I'd also alter the default (mhrCodeDescription) to be the non staff version, and make this version mhrStaffCodeDescription (simpler I think when you're reading it the first time)

Reply:
i.e. a future implementation to have other staff descriptions could be something like [...].includes(code) then code+= 'Staff'

Reply:
Good suggestions.  I have updated code.